src/aiodynamo/http/aiohttp.py

```diff
@@ -41,9 +33,7 @@ class AIOHTTP(HTTP):
             method="GET", url=url, headers=headers, timeout=timeout
         ) as response:
             if response.status >= 400:
-                raise RequestFailed(
-                    url, response.status, await response.read(), headers, None
-                )
+                raise RequestFailed(response.status)
```

Comment:
How does dynamo server signal logical errors, is it not `400` status code?

Reply:
Ahh, sorry I was looking at wrong method, I can see that handled below in `post`.

Reply:
oops shouldn't include status, forgot to remove it here